pkgs/development/python-modules/uqbar/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pip
+, black
+, unidecode
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "uqbar";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-cEhWXGtMSXVjT5QigDedjT/lwYQnVqPCE5vbctXWznk=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    pip
+    black
+    unidecode
+    sphinx
+  ];
+
+  pythonImportsCheck = [ "uqbar" ];
+
+  meta = {
+    description = "Tools for creating Sphinx and Graphviz documentation";
+    license = lib.licenses.mit;
+    homepage = "https://github.com/josiah-wolf-oberholtzer/uqbar";
+    changelog = "https://github.com/josiah-wolf-oberholtzer/uqbar/releases";
```

```suggestion
    changelog = "https://github.com/josiah-wolf-oberholtzer/uqbar/releases/tag/v${version}";
```
The latest tag seems to be wrong...

---

The latest tag is wrong on the github, what should I plan to do about that?

---

Should I still change the changelog URL if it will end up pointing to a dead link?

---

It probably will be fixed in the next release and can be left as is.